c2corg_api/models/document.py

```diff
@@ -62,8 +53,7 @@ def redirects_to(self):
             Integer, ForeignKey(schema + '.documents.document_id'),
             nullable=True)
 
-    quality = Column(
-        Enum(name='quality_type', inherit_schema=True, *quality_types))
+    quality = Column(enums.quality_type)
```

Comment:
Is this change OK @tsauerwein ?
Do we need to have ``inherit_schema=True`` somewhere?